databases/week3/Meal_application.sql

```diff
@@ -0,0 +1,372 @@
+﻿
+-- Create these queries- Meal
+
+create database meal_application;
+use meal_application;
+
+SET NAMES utf8mb4;
+
+
+
+CREATE TABLE `Meal` (
+ 
+`id` int (10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  
+`title` varchar(255) NOT NULL,
+  
+`description` text NOT NULL,
+  
+`location` varchar(255) NOT NULL,
+  
+`when` datetime timestamp,
+  

```

Comment:
this is not accepted and generates error
try this one please ``when` datetime DEFAULT CURRENT_TIMESTAMP,`